skypy/pipeline/tests/test_config.py

```diff
@@ -34,6 +32,12 @@ def test_load_skypy_yaml():
         load_skypy_yaml(filename)
 
 
+def test_empty_ref():
+    filename = get_pkg_data_filename('data/test_empty_ref.yml')
+    with pytest.raises(ValueError, match='empty reference'):
```

Comment:
Looking at the yml file for this test, is this still an empty reference?  Or a broken one?

Reply:
My thinking here was that at the config (i.e. YAML) level we can only catch empty strings being passed when the `!ref` tag is used without argument. We could also let it go to the pipeline level and check there, but then we lose the information about where the error happened.

Which now makes me think that the next iteration of this should be adding 'file', 'line' and 'column' information to `Item` on construction so that the pipeline can do all of the checking and give great error messages.